### PR TITLE
[FIX] Making optional control of precision of annotation occluding

### PR DESCRIPTION
**Pull request by paireks**
This PR includes:
- revert of offset in a shader to make existing behaviour unchanged
- adding parameter for the user to control the offset, so he can decide if he want to make it more precise or not
- 2 new examples with usage of this additional parameter, for which changing of the offset parameter makes occluding better
- update of d.ts file

Looking forward for review.

### CODE REVIEW

File: examples/annotations/annotations_createWithMouse_orthoWithZMarkerOffset.html [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+</head>
+
+<body>
+<input type="checkbox" id="info-button"/>
+<label for="info-button" class="info-button"><i class="far fa-3x fa-question-circle"></i></label>
+<canvas id="myCanvas"></canvas>
+<div class="slideout-sidebar">
+    <img class="info-icon" src="../../assets/images/annotation_icon.png"/>
+    <h1>AnnotationsPlugin</h1>
+    <h2>Click or tap the model to create annotations, control OcclusionTester.js precision</h2>
```
Comment (xeolabs) [inline examples/annotations/annotations_createWithMouse_orthoWithZMarkerOffset.html:94]: Could we rename this to "Click or tap the model to create annotations, with configured marketZOffset"
So as not to mention OcclusionTester.js, which is private code

File: examples/annotations/annotations_createWithMouse_orthoWithZMarkerOffsetZero.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+</head>
+
+<body>
+<input type="checkbox" id="info-button"/>
+<label for="info-button" class="info-button"><i class="far fa-3x fa-question-circle"></i></label>
+<canvas id="myCanvas"></canvas>
+<div class="slideout-sidebar">
+    <img class="info-icon" src="../../assets/images/annotation_icon.png"/>
+    <h1>AnnotationsPlugin</h1>
+    <h2>Click or tap the model to create annotations, control OcclusionTester.js precision</h2>
```
Comment (xeolabs) [inline examples/annotations/annotations_createWithMouse_orthoWithZMarkerOffsetZero.html:94]: Same here

**Review by xeolabs — CHANGES_REQUESTED** [review]

File: examples/annotations/annotations_createWithMouse_orthoWithZMarkerOffset.html (hunk 1 above)
Comment (paireks) [reply examples/annotations/annotations_createWithMouse_orthoWithZMarkerOffset.html]: Done

File: examples/annotations/annotations_createWithMouse_orthoWithZMarkerOffsetZero.html (hunk 2 above)
Comment (paireks) [reply examples/annotations/annotations_createWithMouse_orthoWithZMarkerOffsetZero.html]: Done

**Pull request merged**